heat/core/operations.py

```diff
@@ -255,13 +287,14 @@ def __reduce_op(x, partial_op, reduction_op, **kwargs):
 
     # if local tensor is empty, replace it with the identity element
     if 0 in x.lshape and (axis is None or (x.split in axis)):
-        neutral = kwargs.get("neutral")
         if neutral is None:
             neutral = float("nan")
         neutral_shape = x.lshape[:split] + (1,) + x.lshape[split + 1 :]
         partial = torch.full(neutral_shape, fill_value=neutral, dtype=x._DNDarray__array.dtype)
     else:
         partial = x._DNDarray__array
+
+    #
```

Comment:
dead code? or spacing?

Reply:
resolved, comment I wanted to introduce was missing